### ISSUE TITLE
Baseline Detections False Positives

### ISSUE THREAD

**Issue by goktugyildirim** [waymo-research/waymo-open-dataset#342]
I integrated Waymo Open Dataset to ROS2 for the purpose of visualizing the dataset. I'm planning to use baseline detections to use with our tracker evaluation work, but I think baseline detections contain lots of false positives. You can check out this video: https://www.youtube.com/watch?v=HNwCDacDE2g 

I used segment-4140965781175793864_460_000_480_000 in the test set. The bounding boxes represent the baseline detections having the type of vehicle. Is it an expected result? I have no idea the performance of your detection model.

**Comment by peisun1115** [comment]
yeah, the baseline detectors isn't supposed to be very good. You can use a better detector.
